api/app/models/env_canada.py

```diff
@@ -182,14 +243,17 @@ def mark_prediction_model_run_processed(session: Session,
         month=now.month,
         day=now.day,
         hour=now.hour, tzinfo=datetime.timezone.utc)
-    prediction_run_timestamp = adjust_model_day(prediction_run_timestamp, model_run_hour)
-    prediction_run_timestamp = prediction_run_timestamp.replace(hour=model_run_hour)
+    prediction_run_timestamp = adjust_model_day(
+        prediction_run_timestamp, model_run_hour)
+    prediction_run_timestamp = prediction_run_timestamp.replace(
+        hour=model_run_hour)
```

Comment:
I wonder what happens here hmm. Have you recently changed your lint settings?

Reply:
Haven't changed my settings, no. I think it's pylint & vscode conspiring against me again